src/main/java/seedu/duke/logic/commands/SummaryCommand.java

```diff
@@ -12,7 +12,7 @@
             + "| Set       | set COMPONENT VALUE        | set INPUT BOOLVALUE |\n"
             + "| Add       | add CONFIG COMPONENT VALUE | add INPUT GATE      |\n"
             + "| Calculate | calc EFF_VALUE             | calc                |\n"
-            + "| Exit      | exit                       | exit                |\n"
+            + "| Exit      | bye                        | bye                 |\n"
```

Comment:
Nice work! I admire the beauty of this change.

Reply:
fml